README.md

```diff
@@ -1,36 +1,31 @@
 ![screenshot](screenshot.png)
 
-
 # LD-Relay Locust File
 
-
 This is a locust file meant to emulate the load of launchdarkly server and mobile clients.
 
-
 # Setup
 
 ## Prerequisites
 
-If you are running the testing infrastructure outside of Docker you will need to install the dependencies. This project utilizes `Python 3`, and `Locust 0.13.5`. You can install the required version of Locust with:
+If you are running the testing infrastructure outside of Docker you will need to install the dependencies. This project utilizes `Python 3`, and `Locust 2.10.1`. You can install the required version of Locust with:
```

Comment:
I think `If you are running the testing infrastructure outside of Docker you will need to install the dependencies.`

Should be:
`If you are running the testing infrastructure, outside of Docker, you will need to install the dependencies.`

or
`If you are running the testing infrastructure outside of Docker, you will need to install the dependencies.`

Reply:
Or maybe
`When running outside of docker you will need...`